### PR TITLE
Fix column type on mysql

### PR DESCRIPTION
**Pull request by kesselb**
Length option on mysql/maria does not affect the min/max values the column can store :(

https://stackoverflow.com/questions/5634104/what-is-the-size-of-column-of-int11-in-mysql-in-bytes

https://github.com/nextcloud/3rdparty/blob/51285f0a85f53a534201ea9a189669c2822ce9b8/doctrine/dbal/src/Schema/MySQLSchemaManager.php#L192-L200

### CODE REVIEW

**Review by ChristophWurst — APPROVED** [review]
:exploding_head:

**Comment by ChristophWurst** [comment]
I've deleted the backport comment because the migration only exists for v1.13

File: lib/Migration/Version1130Date20220412111833.php [hunk 1]
```diff
@@ -86,6 +89,9 @@ public function changeSchema(IOutput $output, Closure $schemaClosure, array $opt
 		/** @var ISchemaWrapper $schema */
 		$schema = $schemaClosure();
 
+		// bigint and autoincrement are not possible on sqlite: https://github.com/nextcloud/server/commit/f57e334f8395e3b5c046b6d28480d798453e4866
```
Comment (kesselb) [inline lib/Migration/Version1130Date20220412111833.php:92]:
```suggestion
		// bigint and primary key with autoincrement is not possible on sqlite: https://github.com/nextcloud/server/commit/f57e334f8395e3b5c046b6d28480d798453e4866
```

**Pull request merged**